packages/react/src/primitives/shared/utils.ts

```diff
@@ -56,14 +56,14 @@ export const classNameModifier = (
  * @param base
  * @param modifier
  * @param flag
- * @returns the modified class name or null
+ * @returns the modified class name or empty string
  */
 export const classNameModifierByFlag = (
   base: ComponentClassName,
   modifier: Modifiers,
-  flag: boolean
+  flag?: boolean
 ): string => {
-  return flag ? `${base}--${modifier}` : null;
+  return flag ? `${base}--${modifier}` : '';
```

Comment:
It seemed simpler to just return empty string. I would expect this function to return a string, not null. I did a quick test to ensure that empty string and null are treated the same by `classnames`:
https://stackblitz.com/edit/react-ts-kw4xuh?file=App.tsx,tsconfig.json